packages/samples/headless-ssr-commerce/app/products/[productId]/page.tsx

```diff
@@ -11,19 +12,25 @@ export default async function ProductDescriptionPage({
 }) {
   // Sets the navigator context provider to use the newly created `navigatorContext` before fetching the app static state
   const navigatorContext = new NextJsNavigatorContext(headers());
-  searchEngineDefinition.setNavigatorContextProvider(() => navigatorContext);
+  standaloneEngineDefinition.setNavigatorContextProvider(
+    () => navigatorContext
+  );
 
   // Fetches the static state of the app with initial state (when applicable)
-  const staticState = await searchEngineDefinition.fetchStaticState();
+  const staticState = await standaloneEngineDefinition.fetchStaticState({
+    controllers: {parameterManager: {initialState: {parameters: {}}}},
```

Comment:
Ideally we wouldn't need to do that with the standaloneEngineDefinition... I'll see if I can fix this.